sdk/core/azure-core/src/main/java/com/azure/core/http/policy/RetryPolicy.java

```diff
@@ -142,18 +145,24 @@ private Mono<HttpResponse> attemptAsync(final HttpPipelineCallContext context, f
                     }
                 } else {
                     if (tryCount >= retryStrategy.getMaxRetries()) {
-                        LOGGER.info("Retry attempts have been exhausted after {} attempts.", tryCount);
+                        LOGGER.atVerbose()
+                            .addKeyValue(LoggingKeys.TRY_COUNT_KEY, tryCount)
+                            .log("Retry attempts have been exhausted.");
                     }
                     return Mono.just(httpResponse);
                 }
             })
             .onErrorResume(Exception.class, err -> {
                 if (shouldRetryException(err, tryCount)) {
-                    LOGGER.verbose("[Error Resume] Try count: {}, Error: {}", tryCount, err);
+                    LOGGER.atVerbose()
+                        .addKeyValue(LoggingKeys.TRY_COUNT_KEY, tryCount)
+                            .log("Error resume.", err);
                     return attemptAsync(context, next, originalHttpRequest, tryCount + 1)
                         .delaySubscription(retryStrategy.calculateRetryDelay(tryCount));
                 } else {
-                    LOGGER.info("Retry attempts have been exhausted after {} attempts.", tryCount, err);
+                    LOGGER.atVerbose()
+                        .addKeyValue(LoggingKeys.TRY_COUNT_KEY, tryCount)
+                        .log("Retry attempts have been exhausted.");
```

Comment:
We should log this as info and also we should include the exception (`err`) in the log.
```suggestion
                    LOGGER.atInfo()
                        .addKeyValue(LoggingKeys.TRY_COUNT_KEY, tryCount)
                        .log("Retry attempts have been exhausted.", err);
```